dubbo-rpc/dubbo-rpc-triple/src/main/java/org/apache/dubbo/rpc/protocol/tri/call/ServerStreamServerCallListener.java

```diff
@@ -34,11 +40,19 @@ public void onReturn(Object value) {
     }
 
     @Override
-    public void onMessage(Object message) {
+    public void onMessage(Object message,DefaultHttp2WindowUpdateFrame stream, Http2Connection connection) {
```

Comment:
Can these parameters not be added to the interface?

Reply:
If it is necessary to add DefaultHttp2WindowUpdateFrame, can it be replaced with an interface?